hcf_backend/backend.py

```diff
@@ -265,6 +265,7 @@ def _get_requests_from_hs(self, n_min_requests):
     def delete_read_batches(self):
         if self.consumed_batches_ids:
             self.consumer.delete(self.hcf_consumer_slot, self.consumed_batches_ids)
+            LOG.info('Deleting read batches: %s', self.consumed_batches_ids)
```

Comment:
I wonder if it is not better to LOG this in debug level, not info. What do you think?

Reply:
Yeah I was thinking about that too @kalessin.

However, the main motivation of this would be in production spiders where we need to determine which batches are not yet deleted due to the spider prematurely closing (_like infra issues_).

Having both logs of *reading* and *deleting* of batches would allows us to use the process of elimination to check the undeleted batch IDs.

Also, having the delete confirmation on the `INFO` level would prevent us from receiving all those superfluous `DEBUG` logs from other components in the code. Going thru all the logs via `iter()` would be so much faster without the other logs :)

cc @VMRuiz

Reply:
fair enough.